packages/rest-client/src/services/types/meta.ts

```diff
@@ -2,6 +2,7 @@ export type RequestMetaData = Record<string, any> | undefined
 export type Error = any // todo
 
 export type Meta = MetaLoading | MetaSuccess | MetaError
+export type StatefulMeta = Record<string, Meta>
```

Comment:
i think the type should be named `ContextualMeta` (? or something like that?) since both of these types are techincally hook state